DOCS-5676-4.2-reproducible-builds-links-kt (#962)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/reproducible-builds.adoc

> 

Reviewer: edit of HIDDEN TEXT

= Reproducible Builds
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
You should be able to build a particular version of your Mule project at any point in time. This facilitates maintenance, because the versions of your projects in production are not always the latest ones.

> 

Reviewer: HIDDEN TEXT

= Reproducible Builds
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
Being able to build a particular version of your Mule project at any time facilitates maintenance, because the versions of your projects in production are not always the latest ones.

To help achieve this goal:

> * Use source control for all your Mule projects and apply branching, merging, and tagging to them. (Do not check in Eclipse-specific files.) Additionally, you can check https://git-scm.com/book/en/v2[Git Source Control] documentation for details.

Reviewer: GLOBAL formatting edit for all list items, per style reference:
```suggestion
* Use source control for all your Mule projects and apply branching, merging, and tagging to them.
Do not check in Eclipse-specific files. 
See the https://git-scm.com/book/en/v2[Git Source Control] documentation for details.
```

> * Manage dependencies strictly by using Maven and an in-house repository manager, such as Nexus. See xref:using-maven-with-mule.adoc[Maven Support in Mule] documentation for guidelines.

Reviewer: ```suggestion
* Manage dependencies strictly by using Apache Maven and an in-house repository manager, such as Nexus.
See the xref:using-maven-with-mule.adoc[Maven Support in Mule] documentation for guidelines.
```
